### PR TITLE
Wrap up version 7 changes

### PR DESCRIPTION
**Pull request by ahangarha**
### Summary

Some minor changes before getting Shakapacker 7 shipped. Key changes:

Todos:
- [x] Move `test_app` to `shakapacker` directory
- [x] Reorganize Changelog

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,26 +1,60 @@
...
 - [Testing the generator](#testing-the-generator)
-- [Find existing issues](#find-existing-issues)
 
----
-## Setting Up a Development Environment
+## Reporting Issues
+If you encounter any issues with the project, please first check the exisiting issues (including closed ones). If the issues is not reported before, please opening an issue on our GitHub repository. Please provide a clear and detailed description of the issue, including steps to reproduce it. Creating a demo repository to demonstrate the issue would be ideal (and in some cases necessary).
 
-1. Install [Yarn](https://yarnpkg.com/)
+If looking for contributing to the project by fixing existing issues, we recommend looking at closed issues, particularly with the "[help wanted](https://github.com/shakacode/shakapacker/issues?q=is%3Aissue+label%3A%22help+wanted%22+is%3Aclosed+)" label.
```
Comment (justin808) [inline CONTRIBUTING.md:15]: Maybe mention webpacker as well, but, TBH, issues worth fixing should NOT be marked as closed. So I question this advice. I'd say look at any issues with help wanted, but first open issues.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,26 +1,60 @@
...
+      ```
+   - For npm package, use `yalc` with following steps:
+      ```bash
+      # In Shakapacker root directory
+      yalc publish
+      # In Rails app for testing
+      yalc link shakapacker
+
+      # After every chagnes in shakapacker, run the folloing in Shakapacker directory
+      yalc push # or yalc publish --push
```
Comment (justin808) [inline CONTRIBUTING.md:45]: type changes above line 44

why not always the publish option?

**Review by justin808 — CHANGES_REQUESTED** [review]
Only one small comment to address.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ahangarha) [reply CONTRIBUTING.md:15]: By the way, this text was there. I just moved it up.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ahangarha) [reply CONTRIBUTING.md:45]: Using `yalc push` updates all the projects using the package.

**Comment by justin808** [comment]
@ahangarha  can I merge?

**Comment by ahangarha** [comment]
@justin808 Let's keep this open for minor changes that won't change anything substantial. That was the idea for making this branch and draft PR.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ahangarha) [reply CONTRIBUTING.md:15]: @justin808 May you please check the closed [help wanted](https://github.com/shakacode/shakapacker/issues?q=is%3Aissue+label%3A%22help+wanted%22+is%3Aclosed+) issues and see if we should include any of them in version 7 (as a breaking change)? For example, I just noticed you mentioned #143 to be added in version 7.

**Pull request merged**